padinfo/view/pantheon.py

```diff
@@ -16,10 +17,14 @@
 if TYPE_CHECKING:
     from dadguide.models.monster_model import MonsterModel
 
+MAX_MONS_TO_SHOW = 20
+MAT_TYPES = [0, 12, 14, 15]
```

Comment:
can you use the enum strings here (and everywhere) instead of the numbers? would be a lot clearer

(I know you can't import the enums but like compare to those instead of to the integers)